server/models/oauth2.py

```diff
@@ -40,7 +40,7 @@ async def auth(
 
 
 def create_access_token(
-    *, data: Dict, expires_delta: timedelta = timedelta(minutes=20)
+    *, data: Dict, expires_delta: timedelta = timedelta(minutes=env.TOKEN_EXPIRATION)
```

Comment:
Essa parametrização do tempo de expiração do token é escopo de outra issue.

```suggestion
    *, data: Dict, expires_delta: timedelta = timedelta(minutes=20)
```